Framework/Backend/services/kafka.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+*/
+
+const { Kafka, logLevel } = require('kafkajs')
+const WebSocketMessage = require('../websocket/message.js');
+const log = new (require('./../log/Log.js'))(`${process.env.npm_config_log_label ?? 'web-ui'}/kafka`);
```

Comment:
I think it would be good to stick to one naming convention for the label of the framework
Here I can see `web-ui` is used while in the rest of the services we have `framework`